tests/cnf/ran/talm/internal/tsparams/talmvars.go

```diff
@@ -14,12 +20,21 @@ var (
 
 	// ReporterNamespacesToDump tells to the reporter from where to collect logs.
 	ReporterNamespacesToDump = map[string]string{
-		TestNamespace: "",
+		TestNamespace:      "",
+		TemporaryNamespace: "",
 	}
 
 	// ReporterCRDsToDump tells to the reporter what CRs to dump.
 	ReporterCRDsToDump = []k8sreporter.CRData{
```

Comment:
Let's use this name for spoke since spoke kubeconfig is the default, and then a ReporterHubCRsToDump for hub? 

Btw, it should be CR to dump instead of CRD as CRD is a static definition of what fields a CR can use. CR is the actual config applied.

Reply:
Or maybe D here meant data? -_-

Reply:
@yliu127 I'm not sure since I think I just copied it over during the migration. But either way CR seems more precise and it's what the comment uses anyway